transaction-volume-anomaly-detection/src/agent.js

```diff
@@ -0,0 +1,375 @@
+const {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  getEthersProvider,
+  getTransactionReceipt,
+} = require("forta-agent");
+
+const {
+  getContractsByChainId,
+  globalSensitivity,
+  getBlocktimeByChainId,
+  getMinBucketBlockSizeByChainId,
+} = require("./agent.config");
+
+let { bucketBlockSize } = require("./agent.config");
+
+const ARIMA_SETTINGS = {
+  p: 2,
```

Comment:
how were those derived?

Reply:
also, what parameters would need to change to make this less/more sensitive? May be good to include in the documentation as well in case someone wants to fork this one.

Reply:
These are the optimal ARIMA configs, but am adding a more in-depth explanation about each one of these to the readme so that users can configure better

Reply:
Added the config explanation to the readme